app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -1,6 +1,7 @@
 module VerifySpAttributesConcern
   def needs_completion_screen_reason
     return nil if sp_session[:issuer].blank?
+    return nil if sp_session&.dig(:request_url).blank?
```

Comment:
Re: The nil chaining, the previous line does an unguarded access to `[:issuer]`, so an error would have already been thrown if `sp_session` is `nil`?

Also, any reason to use `dig` over property access, like we do on the previous line?

```suggestion
    return nil if sp_session[:request_url].blank?
```

Reply:
Nope, it was a copy/paste from similar art in https://github.com/18F/identity-idp/blob/0de18496f5d427b98c97e98326186f1b71788fbb/app/controllers/users/authorization_confirmation_controller.rb#L30-L34